SampleApp/SampleApp/Screens/CreatablesView.swift

```diff
@@ -15,20 +15,29 @@ struct CreatablesView: View {
     @ObservedObject private var viewModel = CreatableViewModel()
 
     var body: some View {
-        ScrollView {
-            HStack {
-                TextField("location", text: $viewModel.location)
-                    .roundedBorderTextField()
-                Button("CALL") {
+        HStack(spacing: 16) {
+            Spacer()
+            TextField("location", text: $viewModel.location)
+                .roundedBorderTextField()
+            VStack {
+                // Use case 1: Use viewFatcher to auto generate view after data comes back
+                Button("CALL (Creatables)") {
                     fetchCreatables()
                 }
+                // Use case 2: Fetch data without view fetcher
+                Button("CALL (Data)") {
```

Comment:
This is for QA can test different interface from Sample app.
Note: I'm happy to change to a different name of these buttons!

Reply:
Updated: Change "CALL" to "FETCH"